composer.json

```diff
@@ -152,7 +161,10 @@
         "drupal/search_api": "^1.0",
         "drupal/jsonapi": "^1.1",
         "drupal/openapi": "1.0-alpha1",
-        "drupal/simple_oauth": "^2.0@RC"
+        "drupal/simple_oauth": "^2.0@RC",
+        "oomphinc/composer-installers-extender": "^1.1",
+        "drupal/dropzonejs": "^1.0@alpha",
```

Comment:
Would be nice if we could require a minimum global version of composer (1.5.0) here (like you can for php). But I don't think there is a way to do that. If you require composer/composer it just puts it in vendor and provides a bin.